CRM/Dedupe/Merger.php

```diff
@@ -1164,15 +1108,11 @@ public static function getRowsElementsAndInfo($mainId, $otherId, $checkPermissio
             $label = ts('[x]');
           }
         }
-        elseif ($field == 'individual_prefix' || $field == 'prefix_id') {
+        elseif ($field == 'prefix_id') {
```

Comment:
I tested what params are actually stored at this point & it was always 'prefix_id' so I think it was a legacy format